docs/validations/validations-getting-started.mdx

```diff
@@ -6,37 +6,49 @@ sidebar_label: 'Getting Started with Validations'
 
 ![Screenshot of a scenario](./img/scenario-example.png)
 
-**Validations** are used to assert states within you game. You can create the scenarios you want to verify using our Scenario Builder,
-and can run collections of scenarios in Unity.
+**Validations** are used to assert states within you game. You can create the Scenarios you want to verify using our Scenario Builder,
+and can run collections of Scenarios in Unity.
```

Comment:
This currently isn't true. Probably want to remove this bit for now and we can provide a walkthrough from the Unity-side if/when that's in place